go.mod

```diff
@@ -8,12 +8,11 @@ require (
 	github.com/eclipse/paho.mqtt.golang v1.2.0
 	github.com/fortytw2/leaktest v1.3.0 // indirect
 	github.com/google/go-cmp v0.3.0 // indirect
-	github.com/google/pprof v0.0.0-20190723021845-34ac40c74b70 // indirect
 	github.com/google/uuid v1.1.1
-	github.com/ianlancetaylor/demangle v0.0.0-20181102032728-5e5cf60278f6 // indirect
 	github.com/pkg/errors v0.8.1 // indirect
 	github.com/spf13/cobra v0.0.5
-	golang.org/x/arch v0.0.0-20190312162104-788fe5ffcd8c // indirect
 	golang.org/x/net v0.0.0-20190724013045-ca1201d0de80 // indirect
 	pack.ag/amqp v0.12.0
 )
+
+go 1.13
```

Comment:
It looks like you switched to Go 1.13 … so you also need to update the go version the travis file